### PR TITLE
added attributes used in mongodb.conf to documentation

### PR DESCRIPTION
**Pull request by jontrainor**

### CODE REVIEW

**Comment by josephholsten** [comment]
<3! :+1: :dancer: :ghost:

File: README.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ For examples see the USAGE section below.
 * `mongodb[:mms_agent][:secret_key]` - MMS Agent API Key
 * `mongodb[:mms_agent][:install_dir]` - Location to install the agent
 * `mongodb[:mms_agent][:log_dir]` - Location to write the agent logfile. If this is a relative path, it's relative to where the service is run (via runit), e.g. set to './main'
+* `mongodb[:enable_rest]` - Enable the ReST interface of the webserver
```
Comment (jspiewak) [inline README.md]: nit: I would group these higher up with attributes that apply regardless of replica set or sharding.

Comment (josephholsten) [inline README.md]: yeah, it's probably time for subsections

**Comment by jamesonjlee** [comment]
ideally, these actually should reflect `node['mongodb']['config']` variables at some point.

**Pull request merged**